src/Model/ComputerAnswer.js

```diff
@@ -0,0 +1,44 @@
+const { Random } = require('@woowacourse/mission-utils');
+
+class ComputerAnswer {
+  #value;
+
+  constructor() {
+    this.#createValue();
+  }
+
+  #createValue() {
+    let numbers = [];
+    while (numbers.length < 3) {
+      const num = Random.pickNumberInRange(1, 9);
+      numbers.push(num);
+      numbers = [...new Set(numbers)];
+    }
```

Comment:
```suggestion
    const numbers = new Set();
    while (numbers.size < 3) {
      const num = Random.pickNumberInRange(1, 9);
      numbers.add(num);
    }
    }
```
`Set`을 사용하셨기 때문에 배열 대신 전부 `Set`을 활용하는 것은 어떨까요??

Reply:
네, 맞아요! Set를 활용하여 리팩터링을 다시 했습니다ㅎㅎ 감사합니다!